client/src/components/ElectronBar.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import CloseTwoToneIcon from '@material-ui/icons/CloseTwoTone';
+import RemoveTwoToneIcon from '@material-ui/icons/RemoveTwoTone';
+import FullscreenTwoToneIcon from '@material-ui/icons/FullscreenTwoTone';
+
+const useStyles = makeStyles({
+  dragbar: {
+    "-webkit-app-region": "drag",
+  },
+  closeButton: {
+    width: "40px",
+    height: "100%",
+    float: "right",
+    transition: "0.2s",
+    "-webkit-app-region": "no-drag",
+    '&:hover': {
+      backgroundColor: "red",
+    }
+  },
+  defaultButton: {
+    width: "40px",
+    height: "100%",
+    float: "right",
+    transition: "0.2s",
+    "-webkit-app-region": "no-drag",
+    '&:hover': {
+      backgroundColor: "rgba(255,255,255,0.2)",
+    }
+  },
+  closeIcon: {
+    width: '100%',
+    height: '100%',
+    transition: "0.2s",
+    fill: 'rgba(255,255,255, 0.5)',
+    '&:hover': {
+      fill: "white",
+    }
+  },
+  minimizeIcon: {
+    width: '100%',
+    height: '100%',
+    paddingBottom: "20px",
+    transition: "0.2s",
+    fill: 'rgba(255,255,255, 0.5)',
+    '&:hover': {
+      fill: "white",
+    }
+  },
+  title : {
+    position: "absolute",
+    color: "rgba(255,255,255, 0.5)",
+    fontSize: "12px",
+    fontWeight: "bold",
+    fontFamily: "Roboto",
+    padding: "4px 0px 2px 8px"
+  }
+});
+
+function closeElectronWindow() {
```

Comment:
Clicking on close, minimize, and maximize buttons threw exceptions for me, might be worthwhile to make a task to make them work/not show for web.
![image](https://user-images.githubusercontent.com/70297748/99020434-7d328000-2513-11eb-9dfb-50fc712df581.png)

Reply:
Yeah, they will certainly throw errors if you try to hit the app in the browser. This can be tied into the environment variable issue I created earlier today. It shouldn't be a problem though since we will be using builds to run the app.